packages/color-area/test/color-area.test.ts

```diff
@@ -799,7 +802,7 @@ describe('ColorArea', () => {
         await elementUpdated(el);
 
         let outputColor = el.color as { h: number; s: number; l: number };
-        const variance = 0.00005;
+        const variance = 0.004;
```

Comment:
Internally sp-color-area is managed by `hsv` color space where for a color area `h: hue` value is constant and `s: saturation` value varies along "x" axis and `v: value` varies across "y" axis of sp-color-area. Now when we initialise it's color value in `hsl` format (line 796), the colorController internally converts it to `hsv` which is very very less lossy but still lossy. Then on line 804 we are again fetching the color value in `hsl` format so the colotController again converts the internal `color object` (which is in `hsv`) to `hsl`. Thus these very very minor losses are still losses due to which we have to reduce the variance value. Earlier the tests were passing because sp-color-area was not properly implemented using `hsv` color space.
Now we either have to reduce the variance value(which was also done by westbrook in hus [pr](https://github.com/adobe/spectrum-web-components/pull/2782/files#diff-340d1dc1dd462adc040557d7d18d8013687320cf3dcd98f25415607f6c673e0d) line 705) or we should test this in `hsv` color-space.